code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -180,10 +180,14 @@
 	if(!CI.wearer)
 		fail("Cruciform is not installed.", user, C)
 		return FALSE
+	
+		var/mob/M = CI.wearer
```

Comment:
Extra tab here needs to be removed, causing `M` to be undefined for the higher line statements.

Reply:
Yep, I'm going to either close this and make a new one or just change this one to reflect what was spoken about in discord.